### ISSUE TITLE
Global livestock distributions

### ISSUE THREAD

**Issue by Geethen** [samapriya/awesome-gee-community-datasets#54]
### Dataset Name
Datatset name: Global livestock distributions
Research paper that introduced paper: https://www.nature.com/articles/sdata2018227#Sec8
Website to access the data: https://www.fao.org/livestock-systems/global-distributions/buffaloes/en/

#### Required
License Information: CCO 1.0 :http://creativecommons.org/publicdomain/zero/1.0

Curated by: FAO, Gilbert et al., 2018
Keywords: Livestock

Last updated: 2010

**Comment by harrybmwells** [comment]
Agreed! This would be a fantastic dataset to have available in GEE.
It may be worth noting that this product has been updated to version 4 (GLW 4) since July 2024 and the maps have been produced for the years 2010, 2015, and 2020 for all species (buffalos, cattle, chickens, ducks, horses, goats, pigs, sheep) at 5 minutes of arc resolution (approx. 10 km at the equator). The two exceptions are ducks and horses which do not have maps for 2020.
All data are still available via the original link: https://www.fao.org/livestock-systems/global-distributions/cattle/en/
This site contains links (hyperlinked years on the right of the web page) for data for all species and years. For example:
Maps for cattle for the year 2010: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/GIVQ75
Maps for cattle for the year 2015: https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/LHBICE
Maps for cattle (and all other species except ducks and horses) for the year 2020: https://data.apps.fao.org/catalog//iso/9d1e149b-d63f-4213-978b-317a8eb42d02
Thanks!

**Comment by harrybmwells** [comment]
Thanks! Sorry if i’m missing something, but can’t seem to find this dataset yet.
